applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -33,25 +53,34 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
   const command = new UpdateCommand({
     TableName: TABLE_NAME,
     Key: {
-      id: portfolioId,
+      id: portfolioDraftId,
     },
-    UpdateExpression: "set #portfolioVariable = :x",
+    UpdateExpression: "set #portfolioVariable = :x, updated_at = :y",
```

Comment:
I'd really like for this to still be readable an expression without needing to read the map later. Could we do:

```suggestion
    UpdateExpression: "set #portfolioVariable = :portfolio, updated_at = :now",
```
?